polkadot/zombienet_tests/functional/0011-async-backing-6-seconds-rate.zndsl

```diff
@@ -0,0 +1,20 @@
+Description: Test we are producing blocks at 6 seconds clip
+Network: ./0011-async-backing-6-seconds-rate.toml
+Creds: config
+
+# Check authority status.
+alice: reports node_roles is 4
+
+# Ensure parachains are registered.
+alice: parachain 2000 is registered within 60 seconds
+alice: parachain 2001 is registered within 60 seconds
+
+# Ensure parachains made progress.
+alice: reports substrate_block_height{status="finalized"} is at least 10 within 100 seconds
+
+# This parachains should produce blocks at 6s clip, let's assume an 8s rate, allowing for
+# some slots to be missed on slower machines
+alice: parachain 2000 block height is at least 30 within 240 seconds
+# This should already have produced the needed blocks
+alice: parachain 2001 block height is at least 30 within 6 seconds
```

Comment:
Hmm. I see this already breaking :P 

Can we not ensure that we never see any candidate timeout? I mean on these test networks we should be relative sure that there are no timeouts.

Reply:
https://github.com/paritytech/polkadot-sdk/blob/0c3f029c35ad70f7f35b3ab889fbbf29fe92a8db/polkadot/node/core/av-store/src/lib.rs#L766-L775

We handle there the events already, we could add there a `candidate_timeout` event. Not really the best place, but maybe we could add it somewhere. Or we just scan every block with some piece of JS code for the events.

Reply:
https://polkadot.js.org/docs/api/examples/promise/system-events

Reply:
> Hmm. I see this already breaking :P

I missed some environment variables for the CI, it is passing now.


> Can we not ensure that we never see any candidate timeout? I mean on these test networks we should be relative sure that there are no timeouts.

I'll look into how to add that assertion as well, as a follow up., opened this to track it:
https://github.com/paritytech/polkadot-sdk/issues/3180

Reply:
> I missed some environment variables for the CI, it is passing now.

I had only seen that it fails after writing this comment :P  What I meant is that this will for sure be flaky :D

Reply:
I'll keep an eye on eye on it  2 out of 2 for now.

To be honest I don't understand why it should be flaky producing blocks at 6s rate is one of the property of our system, I understand it can be worse than that if you don't have enough hw or it is contended, but in this case the test is really small so the CI hw should be enough and if it is not we need to make sure we use reasonably proper hw.

I do think it is valuable to be able to test the outside properties of our system rather than just testing no CandidateTimeout happens, that will catch just one reason we don't produce blocks at 6s, but won't catch all the other reasons we can't think of yet.